.travis/cover.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python3
-
```

Comment:
Oh, had completely forgotten about these Python files, good catch. Let's just wait and confirm Travis can still execute/use them correctly.